examples/metamask/package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "@playwright/test": "1.48.2",
     "@synthetixio/synpress": "workspace:*",
+    "@synthetixio/synpress-cache": "workspace:*",
```

Comment:
If we use dependency of dependency, and both of them are internal packages, turbo doesn't refresh the dependencies properly.

Reply:
ok, but if I understand correctly, end user behavior will change and he will need two separate packages now, one for cache. it's breaking change which I would like to avoid.

Reply:
also, again - I think that end user should need main synpress package and nothing else. only additional wallets.